MapleServer2/Data/Static/ItemMetadataStorage.cs

```diff
@@ -1,15 +1,15 @@
 ﻿using System.Collections.Generic;
 using System.IO;
-using Maple2Storage.Types;
 using ProtoBuf;
+using Maple2Storage.Types;
 
 namespace MapleServer2.Data.Static {
     // This is an in-memory storage to help with determining some metadata of items
     public static class ItemMetadataStorage {
         private static readonly Dictionary<int, ItemMetadata> map = new Dictionary<int, ItemMetadata>();
 
         static ItemMetadataStorage() {
-            using FileStream stream = File.OpenRead("Resources/ms2-item-metadata");
+            using FileStream stream = File.OpenRead("Maple2Storage/Resources/ms2-item-metadata");
```

Comment:
Is there a good reason for changing the filepath?

Reply:
Yes, before the working directory was Maple2Storage, this change changes the working directory to the solution directory.